mods/cnc/chrome/gamesave-browser.yaml

```diff
@@ -112,4 +115,4 @@ Container@GAMESAVE_BROWSER_PANEL:
 					Height: 35
 					Text: Save
 					Visible: False
-				TooltipContainer@TOOLTIP_CONTAINER:
+		TooltipContainer@GAMESAVE_TOOLTIP_CONTAINER:
```

Comment:
This should be able to stay as `TOOLTIP_CONTAINER` - the indentation change should be sufficient.

Reply:
It's fine in the menu but it renders below the panel when in-game.